test-executor/src/main/java/com/cognifide/aet/executor/SuiteExecutor.java

```diff
@@ -17,46 +17,44 @@
  */
 package com.cognifide.aet.executor;
 
```

Comment:
Maybe we could leave the imports order for now and do the change for all classes in separate pull request?

Reply:
@wiiitek, thanks for comments and update. Let's keep this order of imports here. I will update imports order in the codebase and propose a consistent formatting in the repo in a few days (and update the docs).